app/views/admin/stats/gender_stats.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= back_link_to %>
+
+<h2><%= t("admin.stats.genders.title") %></h2>
+
+<div class="stats">
+  <div class="row stats-numbers">
+    <div class="small-12 medium-3 column">
+      <p class="featured">
+        <%= t("admin.stats.genders.female") %><br>
+        <span id="female_count" class="number"><%= @users_with_female_gender %></span>
+      </p>
+    </div>
+
+    <div class="small-12 medium-3 column">
+    <p class="featured">
```

Comment:
I think you could fix the identation here.

Reply:
Thanks for help me